# Translations template for python-watcher.
# Copyright (C) 2016 ORGANIZATION
# This file is distributed under the same license as the python-watcher
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: python-watcher 0.26.1.dev13\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2016-04-22 10:31+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.3.2\n"

#: watcher/api/app.py:33
msgid "The port for the watcher API server"
msgstr ""

#: watcher/api/app.py:36
msgid "The listen IP for the watcher API server"
msgstr ""

#: watcher/api/app.py:39
msgid ""
"The maximum number of items returned in a single response from a "
"collection resource"
msgstr ""

#: watcher/api/app.py:43
msgid ""
"Number of workers for Watcher API service. The default is equal to the "
"number of CPUs available if that can be determined, else a default worker"
" count of 1 is returned."
msgstr ""

#: watcher/api/app.py:50
msgid ""
"Enable the integrated stand-alone API to service requests via HTTPS "
"instead of HTTP. If there is a front-end service performing HTTPS "
"offloading from the service, this option should be False; note, you will "
"want to change public API endpoint to represent SSL termination URL with "
"'public_endpoint' option."
msgstr ""

#: watcher/api/controllers/v1/action.py:364
msgid "Cannot create an action directly"
msgstr ""

#: watcher/api/controllers/v1/action.py:388
msgid "Cannot modify an action directly"
msgstr ""

#: watcher/api/controllers/v1/action.py:424
msgid "Cannot delete an action directly"
msgstr ""

#: watcher/api/controllers/v1/action_plan.py:87
#, python-format
msgid "Invalid state: %(state)s"
msgstr ""

#: watcher/api/controllers/v1/action_plan.py:407
#, python-format
msgid "State transition not allowed: (%(initial_state)s -> %(new_state)s)"
msgstr ""

#: watcher/api/controllers/v1/audit.py:365
msgid "The audit template UUID or name specified is invalid"
msgstr ""

#: watcher/api/controllers/v1/types.py:123
#, python-format
msgid "%s is not JSON serializable"
msgstr ""

#: watcher/api/controllers/v1/types.py:159
#, python-format
msgid "Wrong type. Expected '%(type)s', got '%(value)s'"
msgstr ""

#: watcher/api/controllers/v1/types.py:198
#, python-format
msgid "'%s' is an internal attribute and can not be updated"
msgstr ""

#: watcher/api/controllers/v1/types.py:202
#, python-format
msgid "'%s' is a mandatory attribute and can not be removed"
msgstr ""

#: watcher/api/controllers/v1/types.py:207
msgid "'add' and 'replace' operations needs value"
msgstr ""

#: watcher/api/controllers/v1/utils.py:36
msgid "Limit must be positive"
msgstr ""

#: watcher/api/controllers/v1/utils.py:47
#, python-format
msgid "Invalid sort direction: %s. Acceptable values are 'asc' or 'desc'"
msgstr ""

#: watcher/api/controllers/v1/utils.py:58
#, python-format
msgid "Invalid filter: %s"
msgstr ""

#: watcher/api/controllers/v1/utils.py:65
#, python-format
msgid "Adding a new attribute (%s) to the root of  the resource is not allowed"
msgstr ""

#: watcher/api/middleware/auth_token.py:45
msgid "Cannot compile public API routes"
msgstr ""

#: watcher/api/middleware/parsable_error.py:52
#, python-format
msgid "ErrorDocumentMiddleware received an invalid status %s"
msgstr ""

#: watcher/api/middleware/parsable_error.py:79
#, python-format
msgid "Error parsing HTTP response: %s"
msgstr ""

#: watcher/applier/actions/change_nova_service_state.py:90
msgid "The target state is not defined"
msgstr ""

#: watcher/applier/actions/migration.py:71
msgid "The parameter resource_id is invalid."
msgstr ""

#: watcher/applier/actions/migration.py:124
#, python-format
msgid ""
"Unexpected error occured. Migration failed forinstance %s. Leaving "
"instance on previous host."
msgstr ""

#: watcher/applier/actions/migration.py:140
#, python-format
msgid "Migration of type %(migration_type)s is not supported."
msgstr ""

#: watcher/applier/workflow_engine/default.py:129
#, python-format
msgid "The WorkFlow Engine has failed to execute the action %s"
msgstr ""

#: watcher/applier/workflow_engine/default.py:147
#, python-format
msgid "Revert action %s"
msgstr ""

#: watcher/applier/workflow_engine/default.py:153
msgid "Oops! We need disaster recover plan"
msgstr ""

#: watcher/cmd/api.py:46
#, python-format
msgid "serving on 0.0.0.0:%(port)s, view at http://127.0.0.1:%(port)s"
msgstr ""

#: watcher/cmd/api.py:50
#, python-format
msgid "serving on http://%(host)s:%(port)s"
msgstr ""

#: watcher/cmd/applier.py:39 watcher/cmd/decisionengine.py:40
#, python-format
msgid "Starting server in PID %s"
msgstr ""

#: watcher/common/clients.py:29
msgid "Version of Nova API to use in novaclient."
msgstr ""

#: watcher/common/clients.py:34
msgid "Version of Glance API to use in glanceclient."
msgstr ""

#: watcher/common/clients.py:39
msgid "Version of Cinder API to use in cinderclient."
msgstr ""

#: watcher/common/clients.py:44
msgid "Version of Ceilometer API to use in ceilometerclient."
msgstr ""

#: watcher/common/clients.py:50
msgid "Version of Neutron API to use in neutronclient."
msgstr ""

#: watcher/common/exception.py:59
#, python-format
msgid "Unexpected keystone client error occurred: %s"
msgstr ""

#: watcher/common/exception.py:72
msgid "An unknown exception occurred"
msgstr ""

#: watcher/common/exception.py:92
msgid "Exception in string format operation"
msgstr ""

#: watcher/common/exception.py:122
msgid "Not authorized"
msgstr ""

#: watcher/common/exception.py:127
msgid "Operation not permitted"
msgstr ""

#: watcher/common/exception.py:131
msgid "Unacceptable parameters"
msgstr ""

#: watcher/common/exception.py:136
#, python-format
msgid "The %(name)s %(id)s could not be found"
msgstr ""

#: watcher/common/exception.py:140
msgid "Conflict"
msgstr ""

#: watcher/common/exception.py:145
#, python-format
msgid "The %(name)s resource %(id)s could not be found"
msgstr ""

#: watcher/common/exception.py:150
#, python-format
msgid "Expected an uuid or int but received %(identity)s"
msgstr ""

#: watcher/common/exception.py:154
#, python-format
msgid "Goal %(goal)s is not defined in Watcher configuration file"
msgstr ""

#: watcher/common/exception.py:158
#, python-format
msgid "Expected a uuid but received %(uuid)s"
msgstr ""

#: watcher/common/exception.py:162
#, python-format
msgid "Expected a logical name but received %(name)s"
msgstr ""

#: watcher/common/exception.py:166
#, python-format
msgid "Expected a logical name or uuid but received %(name)s"
msgstr ""

#: watcher/common/exception.py:170
#, python-format
msgid "AuditTemplate %(audit_template)s could not be found"
msgstr ""

#: watcher/common/exception.py:174
#, python-format
msgid "An audit_template with UUID %(uuid)s or name %(name)s already exists"
msgstr ""

#: watcher/common/exception.py:179
#, python-format
msgid "AuditTemplate %(audit_template)s is referenced by one or multiple audit"
msgstr ""

#: watcher/common/exception.py:184
#, python-format
msgid "Audit type %(audit_type)s could not be found"
msgstr ""

#: watcher/common/exception.py:188
#, python-format
msgid "Audit %(audit)s could not be found"
msgstr ""

#: watcher/common/exception.py:192
#, python-format
msgid "An audit with UUID %(uuid)s already exists"
msgstr ""

#: watcher/common/exception.py:196
#, python-format
msgid "Audit %(audit)s is referenced by one or multiple action plans"
msgstr ""

#: watcher/common/exception.py:201
#, python-format
msgid "ActionPlan %(action_plan)s could not be found"
msgstr ""

#: watcher/common/exception.py:205
#, python-format
msgid "An action plan with UUID %(uuid)s already exists"
msgstr ""

#: watcher/common/exception.py:209
#, python-format
msgid "Action Plan %(action_plan)s is referenced by one or multiple actions"
msgstr ""

#: watcher/common/exception.py:214
#, python-format
msgid "Action %(action)s could not be found"
msgstr ""

#: watcher/common/exception.py:218
#, python-format
msgid "An action with UUID %(uuid)s already exists"
msgstr ""

#: watcher/common/exception.py:222
#, python-format
msgid "Action plan %(action_plan)s is referenced by one or multiple goals"
msgstr ""

#: watcher/common/exception.py:227
msgid "Filtering actions on both audit and action-plan is prohibited"
msgstr ""

#: watcher/common/exception.py:236
#, python-format
msgid "Couldn't apply patch '%(patch)s'. Reason: %(reason)s"
msgstr ""

#: watcher/common/exception.py:242
#, python-format
msgid "Workflow execution error: %(error)s"
msgstr ""

#: watcher/common/exception.py:246
msgid "Illegal argument"
msgstr ""

#: watcher/common/exception.py:250
msgid "No such metric"
msgstr ""

#: watcher/common/exception.py:254
msgid "No rows were returned"
msgstr ""

#: watcher/common/exception.py:258
#, python-format
msgid "%(client)s connection failed. Reason: %(reason)s"
msgstr ""

#: watcher/common/exception.py:262
msgid "'Keystone API endpoint is missing''"
msgstr ""

#: watcher/common/exception.py:266
msgid "The list of hypervisor(s) in the cluster is empty"
msgstr ""

#: watcher/common/exception.py:270
msgid "The metrics resource collector is not defined"
msgstr ""

#: watcher/common/exception.py:274
msgid "the cluster state is not defined"
msgstr ""

#: watcher/common/exception.py:280
#, python-format
msgid "The instance '%(name)s' is not found"
msgstr ""

#: watcher/common/exception.py:284
msgid "The hypervisor is not found"
msgstr ""

#: watcher/common/exception.py:288
#, python-format
msgid "Error loading plugin '%(name)s'"
msgstr ""

#: watcher/common/exception.py:292
#, python-format
msgid "The identifier '%(name)s' is a reserved word"
msgstr ""

#: watcher/common/exception.py:296
#, python-format
msgid "The %(name)s resource %(id)s is not soft deleted"
msgstr ""

#: watcher/common/exception.py:300
msgid "Limit should be positive"
msgstr ""

#: watcher/common/service.py:36
msgid "Seconds between running periodic tasks."
msgstr ""

#: watcher/common/service.py:39
msgid ""
"Name of this node. This can be an opaque identifier.  It is not "
"necessarily a hostname, FQDN, or IP address. However, the node name must "
"be valid within an AMQP key, and if using ZeroMQ, a valid hostname, FQDN,"
" or IP address."
msgstr ""

#: watcher/common/utils.py:53
#, python-format
msgid ""
"Failed to remove trailing character. Returning original object.Supplied "
"object is not a string: %s,"
msgstr ""

#: watcher/common/messaging/messaging_handler.py:98
msgid "No endpoint defined; can only publish events"
msgstr ""

#: watcher/common/messaging/messaging_handler.py:101
msgid "Messaging configuration error"
msgstr ""

#: watcher/db/purge.py:50
msgid "Audit Templates"
msgstr ""

#: watcher/db/purge.py:51
msgid "Audits"
msgstr ""

#: watcher/db/purge.py:52
msgid "Action Plans"
msgstr ""

#: watcher/db/purge.py:53
msgid "Actions"
msgstr ""

#: watcher/db/purge.py:100
msgid "Total"
msgstr ""

#: watcher/db/purge.py:158
msgid "Audit Template"
msgstr ""

#: watcher/db/purge.py:206
#, python-format
msgid ""
"Orphans found:\n"
"%s"
msgstr ""

#: watcher/db/purge.py:265
#, python-format
msgid "There are %(count)d objects set for deletion. Continue? [y/N]"
msgstr ""

#: watcher/db/purge.py:272
#, python-format
msgid ""
"The number of objects (%(num)s) to delete from the database exceeds the "
"maximum number of objects (%(max_number)s) specified."
msgstr ""

#: watcher/db/purge.py:277
msgid "Do you want to delete objects up to the specified maximum number? [y/N]"
msgstr ""

#: watcher/db/purge.py:340
msgid "Deleting..."
msgstr ""

#: watcher/db/purge.py:346
msgid "Starting purge command"
msgstr ""

#: watcher/db/purge.py:356
msgid " (orphans excluded)"
msgstr ""

#: watcher/db/purge.py:357
msgid " (may include orphans)"
msgstr ""

#: watcher/db/purge.py:360 watcher/db/purge.py:361
#, python-format
msgid "Purge results summary%s:"
msgstr ""

#: watcher/db/purge.py:364
#, python-format
msgid "Here below is a table containing the objects that can be purged%s:"
msgstr ""

#: watcher/db/purge.py:369
msgid "Purge process completed"
msgstr ""

#: watcher/db/sqlalchemy/api.py:361
msgid ""
"Multiple audit templates exist with the same name. Please use the audit "
"template uuid instead"
msgstr ""

#: watcher/db/sqlalchemy/api.py:383
msgid "Cannot overwrite UUID for an existing Audit Template."
msgstr ""

#: watcher/db/sqlalchemy/api.py:494
msgid "Cannot overwrite UUID for an existing Audit."
msgstr ""

#: watcher/db/sqlalchemy/api.py:587
msgid "Cannot overwrite UUID for an existing Action."
msgstr ""

#: watcher/db/sqlalchemy/api.py:698
msgid "Cannot overwrite UUID for an existing Action Plan."
msgstr ""

#: watcher/db/sqlalchemy/migration.py:73
msgid ""
"Watcher database schema is already under version control; use upgrade() "
"instead"
msgstr ""

#: watcher/decision_engine/model/model_root.py:33
#: watcher/decision_engine/model/model_root.py:38
msgid "'obj' argument type is not valid"
msgstr ""

#: watcher/decision_engine/planner/default.py:78
msgid "The action plan is empty"
msgstr ""

#: watcher/decision_engine/strategy/selection/default.py:60
#, python-format
msgid "Incorrect mapping: could not find associated strategy for '%s'"
msgstr ""

#: watcher/decision_engine/strategy/strategies/basic_consolidation.py:288
#: watcher/decision_engine/strategy/strategies/basic_consolidation.py:335
#, python-format
msgid "No values returned by %(resource_id)s for %(metric_name)s"
msgstr ""

#: watcher/decision_engine/strategy/strategies/basic_consolidation.py:448
msgid "Initializing Sercon Consolidation"
msgstr ""

#: watcher/decision_engine/strategy/strategies/basic_consolidation.py:492
msgid "The workloads of the compute nodes of the cluster is zero"
msgstr ""

#: watcher/decision_engine/strategy/strategies/outlet_temp_control.py:147
#, python-format
msgid "%s: no outlet temp data"
msgstr ""

#: watcher/decision_engine/strategy/strategies/outlet_temp_control.py:172
#, python-format
msgid "VM not active, skipped: %s"
msgstr ""

#: watcher/decision_engine/strategy/strategies/outlet_temp_control.py:230
msgid "No hosts under outlet temp threshold found"
msgstr ""

#: watcher/decision_engine/strategy/strategies/outlet_temp_control.py:253
msgid "No proper target host could be found"
msgstr ""

#: watcher/decision_engine/strategy/strategies/vm_workload_consolidation.py:105
#, python-format
msgid "Unexpexted resource state type, state=%(state)s, state_type=%(st)s."
msgstr ""

#: watcher/decision_engine/strategy/strategies/vm_workload_consolidation.py:157
#, python-format
msgid "Cannot live migrate: vm_uuid=%(vm_uuid)s, state=%(vm_state)s."
msgstr ""

#: watcher/decision_engine/strategy/strategies/vm_workload_consolidation.py:241
#, python-format
msgid "No values returned by %(resource_id)s for memory.usage or disk.root.size"
msgstr ""

#: watcher/decision_engine/strategy/strategies/vm_workload_consolidation.py:490
msgid "Executing Smart Strategy"
msgstr ""

#: watcher/objects/base.py:70
#, python-format
msgid "Error setting %(attr)s"
msgstr ""

#: watcher/objects/base.py:108
msgid "Invalid version string"
msgstr ""

#: watcher/objects/base.py:172
#, python-format
msgid "Unable to instantiate unregistered object type %(objtype)s"
msgstr ""

#: watcher/objects/base.py:299
#, python-format
msgid "Cannot load '%(attrname)s' in the base class"
msgstr ""

#: watcher/objects/base.py:308
msgid "Cannot save anything in the base class"
msgstr ""

#: watcher/objects/base.py:340
#, python-format
msgid "%(objname)s object has no attribute '%(attrname)s'"
msgstr ""

#: watcher/objects/base.py:390
#, python-format
msgid "'%(objclass)s' object has no attribute '%(attrname)s'"
msgstr ""

#: watcher/objects/utils.py:40
msgid "A datetime.datetime is required here"
msgstr ""

#: watcher/objects/utils.py:105
#, python-format
msgid "An object of class %s is required here"
msgstr ""

